routes/_pages/settings/general.html

```diff
@@ -27,6 +27,11 @@ <h2>Accessibility</h2>
              bind:checked="$autoplayGifs" on:change="onChange(event)">
       <label for="choice-autoplay-gif">Autoplay GIFs</label>
     </div>
+    <div class="setting-group">
+      <input type="checkbox" id="choice-grouped-images"
+             bind:checked="$groupedImages" on:change="onChange(event)">
+      <label for="choice-groupes-images">Group images</label>
```

Comment:
This `label` doesn't match the `id`. Also I'd probably put this under "preferences" rather than "accessibility"